src/creatures/players/player.hpp

```diff
@@ -198,6 +203,70 @@ class Player final : public Creature, public Cylinder, public Bankable {
 	bool hasAnyMount() const;
 	uint8_t getRandomMountId() const;
 	void dismount();
+
```

Comment:
Perhaps it would be better to create a scoped object for this, similar to what's done with the cyclopedia, player VIP, player titles, etc. This way, we can avoid adding even more variables to the Player class, which is already quite large. For example, OtClientFeatures class?

https://github.com/opentibiabr/canary/blob/main/src/creatures/players/vip/player_vip.cpp
https://github.com/opentibiabr/canary/blob/3ac0bab131658df70151f2c4bfb87564cb213cf3/src/creatures/players/player.hpp#L1272
https://github.com/opentibiabr/canary/blob/3ac0bab131658df70151f2c4bfb87564cb213cf3/src/creatures/players/player.cpp#L75